website/src/styles.scss

```diff
@@ -0,0 +1,457 @@
+$font-stack:    Helvetica, sans-serif;
+$primary-color: #333;
```

Comment:
This is a nice example of a variable here. But for some reason all the colors below are hard-coded.